discovery/factory.go

```diff
@@ -52,12 +52,12 @@ func NewService(
 	signerCreate identity.SignerFactory,
 ) *Discovery {
 	return &Discovery{
-		identityRegistry:            identityRegistry,
-		identityRegistration:        identityRegistration,
-		mysteriumClient:             mysteriumClient,
-		signerCreate:                signerCreate,
-		statusChan:                  make(chan Status),
-		status:                      StatusUndefined,
+		identityRegistry:     identityRegistry,
```

Comment:
This was formatted by goimports in my system. Maybe we have some old version of goimports in the build image?

Reply:
Found it, I was using old version from https://github.com/bradfitz/goimports

Reply:
or not
sorry for these comments fall :(